cg/constants/delivery.py

```diff
@@ -78,9 +78,9 @@
 
 CLINICAL_DELIVERY_TAGS: list[set[str]] = [{HermesFileTag.CLINICAL_DELIVERY}]
 
-FASTQ_ANALYSIS_CASE_TAGS: list[set[str]] = []
+RAW_DATA_ANALYSIS_CASE_TAGS: list[set[str]] = []
 
-FASTQ_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
+RAW_DATA_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
```

Comment:
This is a bit confusing as raw data includes both fastq and bam. Maybe we should treat raw data differently and not include that workflow in this dictionary, would that be feasible? I can't think of any other workaround right now but maybe it is worth discussing

Reply:
Yes it is a design problem that fastq is a workflow and a delivery type I think. Not sure how to address it

Reply:
One idea could be to remove Workflow.FASTQ from `SampleAndCaseDeliveryTagsFetcher`. Since we wan't to stop treating FASTQ/RAW-DATA as a Workflow like the others, I think it would simplify some logic

Reply:
I explained an option below. Not sure why we want to treat WorkFlow.RawData differently, perhaps I missed this. Resolving this for now.

Reply:
I think raw data should at least be unaligned, hence BAM should not  be part of it.